packages/react-native-bridge/CHANGELOG.md

```diff
@@ -2,5 +2,13 @@
 
 ## Unreleased
 
+- [***] Fix launching video preview on Android 11+ [#38377]
+
+
+## 1.70.0
+
 - [**] Fix Android handling of Hebrew and Indonesian translations [#37565]
+
+## 1.70.1
+
```

Comment:
As the PR is already merged I wouldn't apply any correction, but just wanted to share that I think the section shouldn't have been incorporated in this PR as we'll do it when merging the betafix `1.70.1`.

Reply:
Ouch, apologies. The version number mentioned for the `ios 13` item is actually `1.50.0` not `1.70.1` 😞 .

Let me know if you think we should fix that in a separate PR or you are planning to do it as part of the 1.70.1 process indeed @fluiddot , thanks!

Reply:
> Let me know if you think we should fix that in a separate PR or you are planning to do it as part of the 1.70.1 process indeed @fluiddot , thanks!

No worries, I'm planning to fix it as part of the release process ([reference](https://github.com/WordPress/gutenberg/pull/38276/commits/d52ae744c15681ecdac521299c0216b8042de022)).